### DISCUSSION TITLE
DeepinV23Beta在部分情况下无法安装hplip

### DISCUSSION THREAD

**Discussion by Aninana** [linuxdeepin/developer-center#4502] [General | 常规讨论]
HP的打印机部分情况下在V23Beta无法安装驱动，执行 sudo apt install hplip 后报如下错误：
![截图_选择区域_20230526162616](https://github.com/linuxdeepin/developer-center/assets/130546027/998ca84d-bd19-40e2-92e4-96a346d32526)
后来发现通过执行 sudo apt install avahi-daemon printer-driver-postscript-hp sane-utils 就可以解决。
不过也不是一定解决，部分打印机执行后仍然报错，执行 sudo apt install hplip 后：
![截图_选择区域_20230526162726](https://github.com/linuxdeepin/developer-center/assets/130546027/6fce9ca6-30dc-4f05-a97e-1a70de943023)
执行 sudo apt install avahi-daemon printer-driver-postscript-hp sane-utils 后：
![截图_选择区域_20230526162744](https://github.com/linuxdeepin/developer-center/assets/130546027/065f2eac-ed9d-431c-9234-dc4795220f0b)
打印管理器报错：“通过dbus安装驱动失败”
![1](https://github.com/linuxdeepin/developer-center/assets/130546027/798a0a32-080a-4287-89d9-adb144f4fd34)
这个问题在deepin论坛没找到解决方法。

**Comment by BLumia** [comment]
hplip 这个包可能需要打包维护者更新一下。目前写了 python3 < 3.10 的依赖但实际系统提供的 python3 包大于这个版本约束。

**Comment by Zeno-sole** [comment]
转为issues我修一下吧
